src/NHSD.BuyingCatalogue.Documents.API/Startup.cs

```diff
@@ -58,7 +62,22 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                     });
             }
 
-            app.UseHealthChecks("/health");
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapHealthChecks("/health/live",
+                    new HealthCheckOptions
+                    {
+                        Predicate = healthCheckRegistration =>
+                            healthCheckRegistration.Tags.Contains(HealthCheckTags.Live)
+                    });
+
+                endpoints.MapHealthChecks("/health/ready",
+                    new HealthCheckOptions
+                    {
+                        Predicate = healthCheckRegistration =>
+                            healthCheckRegistration.Tags.Contains(HealthCheckTags.Ready)
+                    });
+            });
 
             app.UseRouting()
```

Comment:
The app.UseRouting is duplicated. Can we move the line endpoints.MapControllers(); into the app.UseEndpoints function?